spec/config_spec.rb

```diff
@@ -68,9 +68,10 @@
 
     context "when $XDG_CONFIG_HOME is defined" do
       before do
+        # to check that XDG_CONFIG_HOME takes precedence over ~/.pryrc
```

Comment:
Not sure this comment adds any value. It's too vague and the line below exists to make sure other calls to `File.exist?` don't mangle our results (because something does call that).

Reply:
The comment referred to both of the lines following, not to just the first line following. Anyway, removed it since you don't want it. It's your codebase after all.